build/Version.props

```diff
@@ -2,8 +2,8 @@
   <PropertyGroup>
     <VersionMajor>2</VersionMajor>
     <VersionMinor>0</VersionMinor>
-    <VersionPatch>1</VersionPatch>
-    <ReleaseSuffix Condition=" '$(ReleaseSuffix)' == '' ">servicing</ReleaseSuffix>
+    <VersionPatch>2</VersionPatch>
```

Comment:
should we change this? I think it keep servicing is better

Reply:
We already have 2.0.2-vspre builds out there. And if we put servicing here, that would mean that older builds would come higher in semantic versioning. That's why I kept it as vspre for now. I will later on, move this to 2.0.3, in which case I will make this servicing again.